scripts/install_ostis.sh

```diff
@@ -4,7 +4,6 @@ prepare_platform()
 {
 		cd scripts
 		sudo apt-get update
-		sudo apt-get install nodejs-dev node-gyp libssl1.0-dev
```

Comment:
No longer required here, it is installed later in sc-web as a dependency of npm package